src/app/translations.tsv

```diff
@@ -365,7 +365,9 @@
 "tsv-double-quotes"	"CSV - with tabulation annd double quotes"	"CSV - avec des tabulations et des guillemets"
 "rdf"	"TXT - N-Quads triples"	"TXT - triplets RDF en N-Quads"
 "skos"	"XML - SKOS thesaurus"	"XML - vocabuaire SKOS"
-"json"	"TXT - JSON Array"	"TXT - tableau JSON"
+"json"	"JSON - with an array"	"JSON - avec un tableau"
+"json-lodex"	"JSON - from API Lodex"	"JSON - compatible API Lodex"
+"json-istex"	"JSON - from API ISTEX"	"JSON - compatible API ISTEX"
```

Comment:
I would invert API and Lodex, API and ISTEX in english translation --> "JSON - from Lodex API", "JSON - from ISTEX API"